processes/external-contributions/README.MD

```diff
@@ -0,0 +1,90 @@
+  
+### Review the changes in the PR
+
+- For PRs changing "supported" features, treat it like any other PR coming from within the team.
+  - Remember: we will have to support these changes in the future. Possibly forever!
+- For PRs only making changes to features we don't officially support - be generous. But make sure:
+  - That any new optional functionality is opt-in rather than becoming the default.
+  - That any community features are flagged in documentation and config examples as originating from the community (and thus may not be supported by the team if bugs are found).
+  
+### Run CI against the PR
+
+Note: inspired by the community answer [here](https://github.community/t5/How-to-use-Git-and-GitHub/Checkout-a-branch-from-a-fork/td-p/77).
+
+- Problem: our CI will only run against branches of the `broadinstitute/cromwell` repo submitted by team members.
+- To turn a community contribution into a PR that travis will run against:
+  - Identify a reference to use for the remote branch and check it out. 
+    - Example: for pull request 938 there will be a reference `refs/pull/938/head`
+    - We can fetch that reference to a new branch using: `git fetch -f origin refs/pull/938/head:938_pr_clone`
+  - Push the branch to github
+  - Create a new PR for the clone branch. Indicate that it only exists to test a community contribution. 
+    - Example title: `[PR 938 Clone] PR for CI only`
+
+### Cycle through Review and CI
+
+- If the community contributor makes changes following your reviews or the CI results:
+  - Glance at the changes to make sure they still seem reasonable.
+  - Make any additional comments
+  - Re-fetch the remote reference: `git fetch -f origin refs/pull/860/head:938_pr_clone`
+  - Push the changes back up to github to re-trigger the CI on your clone PR.
+  
+### Merge the PR
+
+- Once the tests have completed successfully and the PR has two approvals, it can be merged.
+- Remember to delete your branch clone PR (and the cloned branch itself too!)
+
+## Shortcuts
+
+### Git Command Shortcut
+
+Note: also inspired by the community answer [here](https://github.community/t5/How-to-use-Git-and-GitHub/Checkout-a-branch-from-a-fork/td-p/77 and the reference gitconfig file [here](https://github.com/lee-dohm/dotfiles/blob/8d3c59004154571578c2b32df2cdebb013517630/gitconfig#L8)).
+
+It's tedious to have to remember the syntax for `git fetch -f origin refs/pull/938/head:938_pr_clone` isn't it? Well
+luckily you don't have to!
+
+**Step 1:** add this line into your `~/.gitconfig` file under the `[alias]` section:
+```
+clone-pr = !sh -c 'git fetch -f origin pull/$1/head:$1_pr_clone && git checkout $1_pr_clone' -
+```
+
+**Step 2:** Your regular `git` command line now has new superpowers:
+```
+[develop] $ git clone-pr 938
+From https://github.com/broadinstitute/cromwell
+ * [new ref]             refs/pull/938/head -> 938_pr_clone
+Switched to branch '938_pr_clone'
+
+[938_pr_clone] $ git push
```

Comment:
I think `git push` working automatically assumes a non-default git setting of some kind. `git push origin HEAD` worked for me.
```
anichols@wm51e-eeb ~/Repos/cromwell (develop) $ git clone-pr 5070
remote: Enumerating objects: 12, done.
remote: Counting objects: 100% (12/12), done.
remote: Total 22 (delta 12), reused 12 (delta 12), pack-reused 10
Unpacking objects: 100% (22/22), done.
From github.com:broadinstitute/cromwell
 * [new ref]             refs/pull/5070/head -> 5070_pr_clone
Switched to branch '5070_pr_clone'
anichols@wm51e-eeb ~/Repos/cromwell (5070_pr_clone) $ git push
fatal: The current branch 5070_pr_clone has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin 5070_pr_clone

```

Reply:
Pretty sweet command though!

Reply:
Added a note below this line. It seems to depend what your `push.default` config value is, so I added a couple of possible workarounds if the `git push` doesn't work